citizen.tests.ts

```diff
@@ -359,4 +360,38 @@ describe("Simnet deployment plan operations", () => {
     // Teardown
     rmSync(tempDir, { recursive: true, force: true });
   });
+
+  it(`the first-class citizenship simnet has the correct balances for the registered accounts`, async () => {
+    // Setup
+    const tempDir = mkdtempSync(join(tmpdir(), "simnet-test-"));
```

Comment:
```suggestion
    const tempDir = mkdtempSync(join(tmpdir(), "simnet-test"));
```
Do we need the last dash?